db/db_test2.cc

```diff
@@ -1701,9 +1701,11 @@ TEST_F(DBTest2, ReadAmpBitmap) {
   }
   delete iter;
 
-  // Read amp is 100% since we read all what we loaded in memory
-  ASSERT_EQ(options.statistics->getTickerCount(READ_AMP_ESTIMATE_USEFUL_BYTES),
-            options.statistics->getTickerCount(READ_AMP_TOTAL_READ_BYTES));
+  // Read amp is on average 100% since we read all what we loaded in memory
+  ASSERT_GE(options.statistics->getTickerCount(READ_AMP_ESTIMATE_USEFUL_BYTES) *
```

Comment:
Can also check that it's not too big, since it can now be greater than 1.

`EXPECT_NEAR(..., 1., .01)`

Also consider adding a test with `read_amp_bytes_per_bit = 1` and `ASSERT_EQ`.